frontend/src/components/AlterSchemaPrepForm/ProjectTenantView.vue

```diff
@@ -226,6 +237,36 @@ watchEffect(() => {
     props.state.deployingTenantDatabaseList = databaseIdList;
   }
 });
+
+useOverrideSubtitle(() => {
```

Comment:
This usage looks a bit confusing. How about using `<slot name='subtitle' />`?

Reply:
We cannot use a slot here since it's overriding its ancestor's (rather than parent's) rendering.